### DISCUSSION TITLE
how do you turn on anti-kick ?

### DISCUSSION THREAD

**Discussion by iamdogshi** [winstxnhdw/lc-hax#475] [Q&A] [answered]
how do you turn on anti-kick ? i`ve pressed / and \ but none of them worked.

**Comment by winstxnhdw** [comment]
I think it’s funny that you even tried `/`.

**Reply by winstxnhdw** [reply]
What were you expecting to happen?

**Reply by iamdogshi** [reply]
i did it in the main menu. the anti-kick just stays as Anti-Kick: Off
![image](https://github.com/winstxnhdw/lc-hax/assets/157415146/403ed367-a909-46d3-a09a-640b5c264cf5)

**Reply by iamdogshi** [reply]
for the anti kick to say on

**Reply by winstxnhdw** [reply]
What’s your keyboard layout? If your physical keyboard is US QWERTY but your digital layout is something else, you either have to find what’s the actual key or you can just switch it back to US.

**Reply by iamdogshi** [reply]
how would i change the digital one?

**Reply by iamdogshi** [reply]
my keyboard is a UK one. the \ is next to left shift but it doesnt work.

**Reply by iamdogshi** [reply]
where would the \ be on a US keyboard?

**Reply by winstxnhdw** [reply, marked as answer]
https://letmegooglethat.com/?q=how+to+change+my+keyboard+layout+on+windows

**Reply by winstxnhdw** [reply]
\ is above Enter on a US keyboard

**Reply by iamdogshi** [reply]
I will try us and see if it works or not. i will let u know if it did or not

**Reply by iamdogshi** [reply]
Switching To A US keyboard worked.

**Reply by winstxnhdw** [reply]
Great. Why did you buy a US keyboard to use a UK layout. That’s funny.

**Reply by iamdogshi** [reply]
i didnt, i used the language in the top result of the search/

**Reply by iamdogshi** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/157415146/4aa76c29-dcba-4647-8573-718b6bdb93ea)
i can switch between the 2 digital keyboards

**Reply by winstxnhdw** [reply]
Just use the one which matches your physical layout.

**Reply by iamdogshi** [reply]
the one that matches my physical layout doesnt work for the anti-kick

**Reply by winstxnhdw** [reply]
That's weird.

**Reply by winstxnhdw** [reply]
I might consider listening directly from the keyboard driver but I don't want the program to be flagged as a keylogger.
